conda_libmamba_solver/utils.py

```diff
@@ -78,16 +79,18 @@ def __enter__(self):
         except UnsupportedOperation:
             log.warning("Cannot capture stream! Bypassing ...", exc_info=True)
         except Exception as exc:
-            traceback.print_exception(type(exc), exc, None, file=sys.stdout)
-            raise
+            if not isinstance(exc, CondaError):
+                traceback.print_exception(type(exc), exc, None, file=sys.stdout)
+            raise exc
 
     def __exit__(self, exc_type, exc_value, tb):
         try:
             self.stop()
         finally:
             if exc_type is not None:
-                traceback.print_exception(exc_type, exc_value, tb, file=sys.stdout)
-                raise exc_type(exc_value)
+                if not isinstance(exc_value, CondaError):
+                    traceback.print_exception(exc_type, exc_value, tb, file=sys.stdout)
+                raise exc_value
```

Comment:
Could you please leave a comment why we're printing the exception here if it's not an error from conda?